operate/qa/integration-tests/src/test/java/io/camunda/operate/it/MigrateProcessInstanceOperationZeebeIT.java

```diff
@@ -701,6 +704,100 @@ public void testMigrateProcessInstanceShouldImportMigratedFlowNodes() throws Exc
     assertThat(migratedFlowNode.getBpmnProcessId()).isEqualTo("demoProcess");
   }
 
+  @Test
+  public void testMigrateSubprocessToSubprocess() throws Exception {
+    // given
+    // process instances that are running
+    final var processDefinitionFrom =
+        tester
+            .deployProcess("migration-subprocess.bpmn")
+            .waitUntil()
+            .processIsDeployed()
+            .getProcessDefinitionKey();
+    final var processFrom =
+        tester
+            .startProcessInstance("prWithSubprocess", null)
+            .and()
+            .waitUntil()
+            .processInstanceIsStarted()
+            .then()
+            .completeTask("taskA")
+            .and()
+            .waitUntil()
+            .flowNodeIsActive("subprocess")
+            .then()
+            .getProcessInstanceKey();
+    final var processDefinitionTo =
+        tester
+            .deployProcess("migration-subprocess2.bpmn")
+            .waitUntil()
+            .processIsDeployed()
+            .getProcessDefinitionKey();
+    // when
+    // execute MIGRATE_PROCESS_INSTANCE operation
+    final var query = createGetProcessInstancesByIdsQuery(List.of(processFrom));
+    final var request =
+        new CreateBatchOperationRequestDto()
+            .setName("migrate process with subprocesses")
+            .setOperationType(OperationType.MIGRATE_PROCESS_INSTANCE)
+            .setQuery(query)
+            .setMigrationPlan(
+                new MigrationPlanDto()
+                    .setTargetProcessDefinitionKey(String.valueOf(processDefinitionTo))
+                    .setMappingInstructions(
+                        List.of(
+                            new MappingInstruction()
+                                .setSourceElementId("taskA")
+                                .setTargetElementId("taskA"),
+                            new MappingInstruction()
+                                .setSourceElementId("subprocess")
+                                .setTargetElementId("subprocess2"),
+                            new MappingInstruction()
+                                .setSourceElementId("innerSubprocess")
+                                .setTargetElementId("innerSubprocess2"),
+                            new MappingInstruction()
+                                .setSourceElementId("taskB")
+                                .setTargetElementId("taskB"))));
+
+    final var mvcResult = postBatchOperation(request, HttpStatus.SC_OK);
+    // and execute the operation
+    tester.waitUntil().operationIsCompleted();
+
+    // then
+    // subprocesses are migrated
+    final var subprocessFlowNodes =
+        searchAllDocuments(flowNodeInstanceTemplate.getAlias(), FlowNodeInstanceEntity.class)
```

Comment:
Are you able to use the searchTerm function in TestRepository to search on multiple unique fields rather than doing a searchAll?